e2etests/package.json

```diff
@@ -54,10 +54,10 @@
     "module_messaging": "mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reports.json test/module_messaging.js",
     "module_twin": "mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reports.json test/module_twin.js",
     "module_methods": "mocha --reporter mocha-multi-reporters --reporter-options configFile=../mocha-reports.json test/module_methods.js",
-    "flaky_tests": "npm-run-all -p -l  job_client twin_disconnect method_disconnect c2d_disconnect d2c_disconnect",
+    "flaky_tests": "npm-run-all -p -l  job_client",
```

Comment:
>job_client [](start = 39, length = 10)

I think job client might be working again.  How long before it becomes wedged again though?

Reply:
I'll run tests separately.